.github/workflows/validation-jobs.yml

```diff
@@ -303,14 +302,14 @@ jobs:
             target/
           key: ${{ runner.os }}-cargo-check-showcase-patches-${{ hashFiles('**/Cargo.toml') }}
       - uses: dtolnay/rust-toolchain@stable
-      - name: Installs cargo-udeps
-        run: cargo install --force cargo-udeps
       - name: Install alsa and udev
         run: sudo apt-get update; sudo apt-get install --no-install-recommends libasound2-dev libudev-dev
       - name: Apply patches
         run: |
+          CODE=0
           for patch in tools/example-showcase/*.patch; do
-            git apply --ignore-whitespace $patch
+            git apply --ignore-whitespace $patch || { echo "::error::$patch failed to apply."; CODE=1; }
```

Comment:
```suggestion
            # Try applying the patch, logging an error if it fails.
            git apply --ignore-whitespace $patch || { echo "::error::$patch failed to apply."; CODE=1; }
```

Reply:
Did you mean to mark this as resolved? I think the force-push may have messed up the review.

Either way, I think the `|| { ... }` syntax could use a comment explaining what it does, for future reviewers.

Reply:
Oops, nuked it in the rebase.

Reply:
> Oops, nuked it in the rebase.

No worries :)